NAMESPACE

```diff
@@ -17,4 +17,33 @@ exportMethods(getLinkDSLD)
 exportMethods(getLinkTTG)
 exportMethods(initialize)
 exportMethods(merge)
+importFrom(assertthat,"TRUE")
+importFrom(assertthat,"for")
+importFrom(assertthat,"function")
+importFrom(assertthat,"if")
+importFrom(assertthat,A)
+importFrom(assertthat,Is)
+importFrom(assertthat,Used)
+importFrom(assertthat,a)
+importFrom(assertthat,address)
+importFrom(assertthat,as)
 importFrom(assertthat,assert_that)
+importFrom(assertthat,check)
+importFrom(assertthat,comings)
+importFrom(assertthat,directory)
+importFrom(assertthat,file)
+importFrom(assertthat,file.exists)
+importFrom(assertthat,help)
+importFrom(assertthat,is)
+importFrom(assertthat,not.)
+importFrom(assertthat,of)
+importFrom(assertthat,or)
+importFrom(assertthat,return)
+importFrom(assertthat,short)
+importFrom(assertthat,string)
+importFrom(assertthat,that)
+importFrom(assertthat,to)
+importFrom(assertthat,utility)
+importFrom(assertthat,valid)
+importFrom(assertthat,well)
+importFrom(assertthat,will)
```

Comment:
@charksong , Looks like something is going wrong with your documentation as it appears to have added a whole sentence to the NAMESPACE file rather than specific functions.  At least I'm pretty sure there is no `will` function coming from the `assertthat` library :)